packages/grafana-toolkit/src/config/webpack.plugin.config.ts

```diff
@@ -10,11 +10,15 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 import * as webpack from 'webpack';
 import { getStyleLoaders, getStylesheetEntries, getFileLoaders } from './webpack/loaders';
 
-interface WebpackConfigurationOptions {
+export interface WebpackConfigurationOptions {
   watch?: boolean;
   production?: boolean;
 }
 type WebpackConfigurationGetter = (options: WebpackConfigurationOptions) => webpack.Configuration;
+export type CustomWebpackConfigurationGetter = (
+  originalConfig: webpack.Configuration,
+  options: WebpackConfigurationOptions
+) => webpack.Configuration;
 
 export const findModuleFiles = (base: string, files?: string[], result?: string[]) => {
   files = files || fs.readdirSync(base);
```

Comment:
Is it possible to use the async version of this function instead?

Reply:
This would make sense if we have any tasks after the webpack task. 

This would require to have the complete loading async and therefore add a lot of noise to this PR.
If you want I can still change everything to async/await.

Reply:
It's probably not critical for this PR, but async functions are a Node.js best practice.